Create ui-c8y-1021-41-0-spliting-cookie-and-basic-auth-7888-graft-release-cd-8335.md (#2950)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-41-0-spliting-cookie-and-basic-auth-7888-graft-release-cd-8335.md

---
date: ""

> title: Basic authentication has been removed during the login process using OAI-Secure

Reviewer: ```suggestion
title: Enhanced login process using OAI-Secure
```

---
date: ""

> title: Basic authentication has been eliminated during the login process using OAI-SECURE.

Reviewer: ```suggestion
title: Basic authentication has been removed during the login process using OAI-Secure
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61832
version: 1021.41.0
---

> During the login process using OAI-SECURE, the UI relied on basic authentication due to logic based on the /currentTenant endpoint. As part of this task, basic authentication has been eliminated from the OAI-SECURE login process.

Reviewer: ```suggestion
During the login process using OAI-Secure, the UI relied on basic authentication due to logic based on the `/currentTenant` endpoint. As part of this change, basic authentication has been eliminated from the OAI-Secure login process.
```

> During the login process using OAI-Secure, the UI relied on basic authentication due to logic based on the `/currentTenant` endpoint. As part of this change, basic authentication has been eliminated from the OAI-Secure login process.

Reviewer 2: OAI Secure has been enhanced by removing the need to call /currentTenant as part of the login. This is of interest to our customers who have written their own login interface. If you are unsure if the new approach is supported on your instance, check for the login option 'tfaSupported' which will indicate you no longer have to use /currentTenant.

Reviewer: Thanks @JanePorter. Please check and approve

> During the login process using OAI-Secure, the UI relied on basic authentication due to logic based on the `/currentTenant` endpoint. As part of this change, basic authentication has been eliminated from the OAI-Secure login process.

Reviewer: ```suggestion
OAI Secure has been enhanced by removing the need to call `/currentTenant` as part of the login. This is of interest to our customers who have written their own login interface. If you are unsure if the new approach is supported on your instance, check for the login option `tfaSupported` which indicates you no longer have to use `/currentTenant`.
```

> The new login process is available for OAI-SECURE where the tfaSupported flag is enabled. This flag was introduced to ensure backward compatibility.

Reviewer: ```suggestion
The new login process is available for OAI-Secure where the `tfaSupported` flag is enabled. This flag was introduced to ensure backward compatibility.
```

> The new login process is available for OAI-Secure where the `tfaSupported` flag is enabled. This flag was introduced to ensure backward compatibility.

Reviewer: ```suggestion
```
